bundles/org.openhab.binding.senechome/README.md

```diff
@@ -0,0 +1,52 @@
+# Senec Home Binding
+
+Binding to request status information from Senec Home Batteries to allow home automation decisions based on your current energy management.
+
+If your power feed is limited by regulations for example, you can switch on devices if a limited rate was applied by your Senec Home device.
+In addition you can switch off devices if the power consumption is getting higher.
+
+Examples: Lights, pool filters, wash machines, ...
+
+## Supported Things
+
+| Thing type id        | Name                                       |
+|----------------------|--------------------------------------------|
+| senechome            | Senec Home Lithium Battery, V2.0 and V2.1  |
+
+
+## Thing Configuration
+
+demo.things
+
+```
+Thing senechome:senechome:pvbattery [ hostname="192.168.0.128" ]
+```
+
+If the thing goes online then the connection to the web interface is successful.
+In case it is offline you should see an error message.
+The refresh interval is 15 seconds by default, but you can optionally provide your own value adding the `refreshInterval` key.
+
+The property `limitationTresholdValue` is used as threshold for channel `powerLimitationState`. It is combined with property `limitationDuration` (seconds) to define a stable status for power limitation. Therefor use `powerLimitationState` to trigger Events (switches) based on your current power limitation.
+
+## Channels
+
+-   powerLimitation, percent, How much is your pv generator limited (0% if not limited anyway)
```

Comment:
Can you make this a table as used with other bindings. You can use markdowntable.com to format the table.

Also add the channel types in the table. Instead of the 2nd argument you have here.